bin/varnishd/cache/cache_obj.c

```diff
@@ -220,15 +222,31 @@ void
 ObjExtend(struct worker *wrk, struct objcore *oc, ssize_t l, int final)
 {
 	const struct obj_methods *om = obj_getmethods(oc);
+	AN(om->objextend);
 
 	CHECK_OBJ_NOTNULL(wrk, WORKER_MAGIC);
 	CHECK_OBJ_NOTNULL(oc->boc, BOC_MAGIC);
 	(void)final;
 	assert(l >= 0);
 
 	Lck_Lock(&oc->boc->mtx);
-	AN(om->objextend);
 	if (l > 0) {
+		/* Pass readahead applies when only sending to one client */
```

Comment:
how could more than one client read an `OC_F_PRIVATE` object?

Reply:
The comment seems correct to me

Reply:
I thought it was causing more confusion than do any good, but nevermind